### ISSUE TITLE
[Bug]: Input OTP blocked by password manager

### ISSUE THREAD

**Issue by julianriise** [shadcn-ui/ui#2939]
### Describe the bug

When typing in the digits, 1Password overlaps the last digit.
![image](https://github.com/shadcn-ui/ui/assets/77626314/00873b3e-8095-4e43-befe-34f85a2900e7)

### Affected component/components

Input OTP

### How to reproduce

1. Open 1Password on Mac
2. Go to https://ui.shadcn.com/docs/components/input-otp
3. Type in the digits in the example component

### System Info

```bash
Arc Browser, MacOS
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by guilhermerodz** [comment]
Hi @julianriise. I think it's not a good choice for 1Password to place an overlay over inputs. Happily, they offer an option to avoid this by adding `data-1p-ignore` to the `<input ... />` element. I'm testing this out and will get back to you.

**Comment by guilhermerodz** [comment]
Confirmed. Try adding `data-1p-ignore` to your input and the password manager's extension input overlay should be hidden.

https://github.com/shadcn-ui/ui/assets/10366880/d79aaf72-dff5-439d-865c-aaf1bdde932c

Regarding the underlying library/primitive `input-otp`: I don't know if I should make `data-1p-ignore` it a default? What do you think @shadcn and @julianriise?

**Comment by guilhermerodz** [comment]
What if I ship a v1.0.2 update that blocks PMs by default, and then add an opt-out prop?

![image](https://github.com/shadcn-ui/ui/assets/10366880/d3306000-401e-4b52-b00b-20f9ded33fcb)

**Comment by julianriise** [comment]
Yeah, that seems like an excellent solution👍🏼

**Comment by guilhermerodz** [comment]
guilhermerodz/input-otp@8f4e92fa777dc7282fcfbe4b81dbbf632038fb31

**Solved for nearly all password managers!** 🎉

- automatic support for Lastpass, 1Password, Bitwarden and Dashlane.
- auto-detection for others PWMs (works majority of the time)
- detects badge collisions and fallbacks to original position when needed

![image](https://github.com/shadcn-ui/ui/assets/10366880/8078a92a-51dc-4dfd-8e73-c455da8416e9)

---

please run
```bash
npm install input-otp@latest
```

as of v1.1.0, there is also a new (optional) prop for input-otp:

```ts
pushPasswordManagerStrategy?:
    | 'increase-width'
    | 'none'
    | 'experimental-no-flickering'
```

**Comment by guilhermerodz** [comment]
hi @julianriise is it solved by now?

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
